MiniApp/Classes/core/Models/MiniAppManifest.swift

```diff
@@ -31,6 +32,32 @@ internal struct MACustomPermissionsResponse: Decodable {
     }
 }
 
+public struct MASDKAccessTokenPermission: Codable, Equatable, Hashable {
```

Comment:
`MASDKAccessTokenPermission` -> I guess this could create some confusion, what do you think? May be `MASDKAccessTokenScopes` / `MASDKAccessTokenMetaData` / `MASDKAccessTokenModel` ?

Reply:
I have a preference for the Scopes one, but the MetaData entry is called `AccessTokenPermission`. What do you think @corycaywood ?

Reply:
Maybe it's just missing an "s": `MASDKAccessTokenPermissions`?

The key name in the manifest is `accessTokenPermissions`, so it seems a little confusing if the name doesn't match.

Reply:
Oh, sorry I just noticed this is also being used in `MiniAppUserInfoDelegate.getAccessToken`. Yeah, I think it makes more sense to change it to `MASDKAccessTokenScopes` because it's being used by the Mini App to specify which scopes they want.

However, in the case that it's used as an array `[MASDKAccessTokenPermission]`  (such as above in this class), it should keep the name `accessTokenPermissions`.